package.json

```diff
@@ -30,6 +30,10 @@
     "@types/requirejs": "^2.1.30",
     "@types/sinon": "^2.3.3",
     "@types/sinon-chai": "^2.7.28",
```

Comment:
when I run npm package after npm install, I get a non-fatal error:
npm ERR! peer dep missing: chai@^3.5.0, required by karma-sinon-chai@1.3.1
(yarn doesn't seem to build it at all anymore, deleting the lib folder contents like it does with WidgetCards)

Reply:
right, this is not compatible with yarn anymore, the differences between NPM and yarn are getting too large to maintain compatibility for both, so I opted for using the more widely adopted NPM. The missing dep is a known issue, I wonder if they have fixed it and I only need to update the packages...